drakcore/drakcore/app.py

```diff
@@ -26,6 +26,17 @@
               pathlib.Path(__file__).parent / "migrations")
 
 
+@app.before_first_request
+def update_metadata_cache():
+    """ Scans whole MinIO bucket and fetch missing metadata files """
+    for analysis in AnalysisProxy(minio, None).enumerate():
```

Comment:
The `enumerate()` is kinda sketchy, as it clashes in my head with python's `enumerate` function

Reply:
:thinking: i kinda agreee. this should probably be reworded